src/control/ToolEnums.cpp

```diff
@@ -141,8 +141,8 @@ auto toolTypeFromString(const string& type) -> ToolType {
     if (type == "eraser") {
         return TOOL_ERASER;
     }
-    if (type == "hilighter") {
```

Comment:
Leave this case in to fix following breaking change.

Reply:
Could we allow for both spellings during Input and only use the new/correct spelling as output?
This way we would have a smooth transition for old users and new users would already use the corrected spelling.

Reply:
Yes that is what I meant with my comment.